pinot-segment-local/src/main/java/org/apache/pinot/segment/local/io/writer/impl/BaseChunkForwardIndexWriter.java

```diff
@@ -186,7 +186,8 @@ protected void writeChunk() {
     }
 
     if (_headerEntryChunkOffsetSize == Integer.BYTES) {
-      Preconditions.checkState(_dataOffset <= Integer.MAX_VALUE, "Integer overflow detected");
+      Preconditions.checkState(_dataOffset <= Integer.MAX_VALUE, "Integer overflow detected. "
+          + "Try to use raw version 3 or 4, reduce targetDocsPerChunk or targetMaxChunkSize");
```

Comment:
The older message (Integer overflow detected) is improved here by adding some tips on how to fix it. These attributes are documented in https://docs.pinot.apache.org/basics/indexing/forward-index#raw-value-forward-index